### ISSUE TITLE
[Revise subclass]     Fanconi anemia

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/mondo#5687]
**Mondo term (ID and Label)**
[Fanconi anemia](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0019391#)

**Suggested revision and reasons**
Fanconi does not always have all of the associaed symptoms, so it is not always true that it is a  [congenital limb malformation](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0019391#)

I do not like the term "congenital hematological disorder" because most of the diseases are genetic but not all necessarily manifest at birth. Can we rename this to Mendelian hematological disorder?
thougts @cmungall @ahamosh  ?

**Comment by pnrobinson** [comment]
Same ting applies to "congenital myopathy"

**Comment by cmungall** [comment]
I agree on both - I made an issue to tackle all congenital-X en masse #5695

**Comment by ahamosh** [comment]
Can we talk about this tomorrow?  I am confused.  Are you saying that you want to rename Fanconi Anemia? I totally agree that the anemia part of Fanconi Anemia is not often or even usually congenital.  I feel differently about congenital myopathy because the congenital myopathies do present at birth (or they aren't congenital myopathies).  Let's discuss tomorrow.
